common/src/main/java/io/netty/util/ResourceLeakDetector.java

```diff
@@ -119,7 +121,7 @@ public static void setEnabled(boolean enabled) {
      * Returns {@code true} if resource leak detection is enabled.
      */
     public static boolean isEnabled() {
-        return getLevel().ordinal() > Level.DISABLED.ordinal();
```

Comment:
@Scottmitch good catch! I think we can't merge this as it is... Also yes I would also like to see a benchmark that shows this.